website/docs/r/cloudwatch_source.html.markdown

```diff
@@ -0,0 +1,117 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_cloudwatch_source"
+description: |-
+  Provides a Sumologic CloudWatch source.
+---
+
+# sumologic_cloudwatch_source
+Provides a [Sumologic CloudWatch source][2].
+
+__IMPORTANT:__ The AWS credentials are stored in plain-text in the state. This is a potential security issue.
+
+## Example Usage
+```hcl
+locals {
+  filters = [{
+    name        = "Exclude Comments"
+    filter_type = "Exclude"
+    regexp      = "#.*"
+  }] 
+  tagfilters = [{
+          "type" = "TagFilters"
+          "namespace" = "All"
+          "tags" = ["k3=v3"]
+        },{
+          "type" = "TagFilters"
+          "namespace" = "AWS/Route53"
+          "tags" = ["k1=v1"]
+        },{
+          "type" = "TagFilters"
+          "namespace" = "AWS/S3"
+          "tags" = ["k2=v2"]
+        }]
+}
+
+resource "sumologic_cloudwatch_source" "terraform_cloudwatch_source" {
+  name          = "CloudWatch Metrics"
+  description   = "My description"
+  category      = "aws/terraform_cw"
+  content_type  = "AwsCloudWatch"
+  scan_interval = 300000
+  paused        = false
+  collector_id  = "${sumologic_collector.collector.id}"
+
+  authentication {
+    type = "AWSRoleBasedAuthentication"
+    role_arn = "arn:aws:iam::604066827510:role/cw-role-SumoRole-4AOLS73TGKYI"
```

Comment:
is this from a real IAM role? can be use a bogus value instead `arn:aws:iam::012345678910:role/sumo-role`